pkg/test/resourcefixture/testdata/basic/gkehub/v1beta1/gkehubfeaturemembership/meshgkehubfeaturemembership/_generated_object_meshgkehubfeaturemembership.golden.yaml

```diff
@@ -20,15 +20,14 @@ spec:
   membershipRef:
     name: gkehubmembership-mesh-${uniqueId}
   mesh:
-    controlPlane: CONTROL_PLANE_MANAGEMENT_UNSPECIFIED
```

Comment:
Okay, this is what i"m looking for.  In pkg/test/resourcefixture/testdata/basic/gkehub/v1beta1/gkehubfeaturemembership/meshgkehubfeaturemembership/_generated_object_meshgkehubfeaturemembership.golden.yaml the mesh.controlPlane was removed. Maybe because the previous golden object was not correctly generated?